baseapp/baseapp_test.go

```diff
@@ -39,39 +39,32 @@ var (
 
 type paramStore struct {
 	db *dbm.MemDB
+	t  *testing.T
```

Comment:
What's the objective of this? You are not testing baseapp here, you are testing here that `MemDB` actually works.

Reply:
Some of the operations may return error, so I want to catch them, doing it here is more friendly for a user

Reply:
It might be more user friendly (still unsure about this though) yet it does not bring clear benefits. Again, we are not testing `MemDB` in this package. We are testing baseapp functionalities.

Reply:
> yet it does not bring clear benefits.

Catching an error is a benefit. A test which blindly ignores an error is negligent.

Reply:
If the db does not do its job correctly, the test would fail nonetheless. Anyways, this is not a huge deal - you can keep it

Reply:
I think it's better to detect that error in the places where it fails rather than in some other place and then spent hours looking for a reason why a test fails.

Reply:
Like I said, it's not a big deal. Let's move on.